### PR TITLE
join us styles

### PR DESCRIPTION
**Pull request by kavabata**
Screenshot 2024-06-01 at 22 45 38

### CODE REVIEW

File: lang/en.php [hunk 1]
```diff
@@ -81,12 +81,11 @@
       'gallery' => 'home'
     ],
     'join' => [
-      'title' => 'JOIN <span>US</span>',
```
Comment (tomasztaw) [inline lang/en.php:84]: Is a (span) needed in this place? I deleted it in "pl".

File: lang/ua.php [hunk 2]
```diff
@@ -81,12 +81,11 @@
       'gallery' => 'home'
     ],
     'join' => [
-      'title' => 'ПРИЄДНУЙТЕСЯ <span>ДО НАС</span>',
```
Comment (tomasztaw) [inline lang/ua.php:84]: same as above - (span)

File: lang/en.php (hunk 1 above)
Comment (kavabata) [reply lang/en.php:84]: span was removed
p.s. 
red is removed, green is added

**Review by tomasztaw — APPROVED** [review]
LGTM

**Pull request merged**